python/lsst/cp/pipe/ptc.py

```diff
@@ -853,37 +783,29 @@ def _boundsForAstier(initialPars, lowers=[], uppers=[]):
     def _getInitialGoodPoints(means, variances, maxDeviationPositive, maxDeviationNegative,
                               minMeanRatioTest, minVarPivotSearch):
         """Return a boolean array to mask bad points.
-
         Parameters
         ----------
         means : `numpy.array`
             Input array with mean signal values.
-
         variances : `numpy.array`
             Input array with variances at each mean value.
-
         maxDeviationPositive : `float`
             Maximum deviation from being constant for the variance/mean
             ratio, in the positive direction.
-
         maxDeviationNegative : `float`
             Maximum deviation from being constant for the variance/mean
             ratio, in the negative direction.
-
         minMeanRatioTest : `float`
             Minimum signal value (in ADU) after which to start examining
             the ratios var/mean.
-
        minVarPivotSearch : `float`
             Minimum variance point (in ADU^2) after which the pivot point
             wher the variance starts decreasing should be sought.
-
```

Comment:
I think you still need blank lines between sections like Parameters, Return and notes